packages/next/server/render.tsx

```diff
@@ -98,8 +92,10 @@ if (process.env.NEXT_RUNTIME !== 'edge') {
   require('./node-polyfill-web-streams')
   tryGetPreviewData = require('./api-utils/node').tryGetPreviewData
   warn = require('../build/output/log').warn
+  postProcessHTML = require('./post-process').postProcessHTML
```

Comment:
Are we able to remove the `NEXT_RUNTIME` env checks here or will that need to wait until we move rendering over to `app-render` in https://github.com/vercel/next.js/pull/38817?

Reply:
I think we'll have to keep it here because unlike RSC, Edge SSR is still expected to work with `pages/`. Maybe we'll find better way to organize the `require`s here.